lib/trino-metastore/src/main/java/io/trino/metastore/StorageFormat.java

```diff
@@ -66,6 +68,14 @@ public String getOutputFormat()
         return outputFormat;
     }
 
+    public String getFileExtension()
+    {
+        if (fileExtension == null) {
+            throw new TrinoException(HIVE_UNSUPPORTED_FORMAT, "File extension is not present in StorageFormat");
```

Comment:
File extension is missing from StorageFormat

Reply:
I am following the get method in this class. updated the `File extension` to `fileExtension`